src/utils/tokens.ts

```diff
@@ -360,8 +360,9 @@ const tokens = {
     }
 
     if (foundAmount < depositAmount) {
+      const availableAmount = selectedUtxos.available;
```

Comment:
This should never happen, but it is a fail-safe to avoid showing to the user `undefined available` tokens

```suggestion
      const availableAmount = selectedUtxos.available ?? 0;
```